ccloud/docs/beginner-cloud.rst

```diff
@@ -7,25 +7,44 @@ Tuturial: |ccloud| CLI
 Overview
 --------
 
-This tuturial will show you how to use the `Confluent Cloud CLI
-<https://docs.confluent.io/current/cloud/cli/install.html#ccloud-install-cli?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
-to interact with your `Confluent Cloud
-<https://confluent.cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
-cluster.
+This tutorial will show you how to use the `Confluent Cloud CLI
+<https://docs.confluent.io/current/cloud/cli/install.html>`__ to interact with
+your `Confluent Cloud <https://confluent.cloud/login>`__ cluster.
+
+After running `start.sh <start.sh>`__, you can complete the following workflow
```

Comment:
The following bullet list describes what start.sh does, not what is completed afterwards

Reply:
Awesome!

Reply:
Sweet!! Thank you!

Reply:
Updated